src/schemas/v2/combined.json

```diff
@@ -36814,6 +36814,182 @@
               }
             }
           }
+        },
```

Comment:
I'm not sure if this should live inside a `propertyPack` or at the top level, as it's not strictly part of this transaction

Reply:
moved out of propertyPack